src/shared/ui/steps/Steps.tsx

```diff
@@ -6,15 +6,19 @@ import { StepsProps } from '.';
 
 import csx from './Steps.scss';
 
-export const Steps = ({ activeStep, steps, onStepClick }: StepsProps) => {
+export const Steps = ({ steps, onChange }: StepsProps) => {
+  const handleStepClick = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+    onChange(+e.currentTarget.getAttribute('data-idx'));
+  };
```

Comment:
I haven't used that before, but I guess currentTarget.getAttribute('data-idx') allows you to get the value that is passed to the html attribute (data-idx in div)?